Index: app/src/main/java/com/example/webdoctruyen/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/webdoctruyen/MainActivity.java b/app/src/main/java/com/example/webdoctruyen/MainActivity.java
--- a/app/src/main/java/com/example/webdoctruyen/MainActivity.java	
+++ b/app/src/main/java/com/example/webdoctruyen/MainActivity.java	(date 1703097523645)
@@ -15,6 +15,6 @@
 
         webView=(WebView) findViewById(R.id.webView);
         webView.getSettings().setJavaScriptEnabled(true);
-        webView.loadUrl("D:\\android_studio\\code\\webdoctruyen\\app\\src\\main\\assets");
+        webView.loadUrl("file:///android_asset/index.html");
     }
 }
\ No newline at end of file
